src/dist-check/src/test/java/org/apache/jmeter/junit/JMeterTest.java

```diff
@@ -113,6 +115,20 @@ public JMeterTest(String testName, JMeterGUIComponent gc) {
         guiItem = gc;
     }
 
+    @Override
+    public Description getDescription() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(getName());
+        if (guiItem instanceof TestBeanGUI) {
+            sb.append(" ").append(guiItem.toString());
```

Comment:
Could this repeated `.append(" ")` be added before the `if`s?

Reply:
This commit is not part of this PR. This PR builds on top of https://github.com/apache/jmeter/pull/496, so please comment JUnit5 migration there.